lib/package-manager.coffee

```diff
@@ -4,6 +4,8 @@ _ = require 'underscore-plus'
 Q = require 'q'
 semver = require 'semver'
 url = require 'url'
+apm = require 'apm'
```

Comment:
Wow, I didn't know this was possible, it definitely isn't intended to be on the require path.

This apm is meant for standalone use only since it is compiled as a command line utility and so its native modules are built against node and not atom shell.